velox/common/encode/tests/Base64Test.cpp

```diff
@@ -86,4 +89,117 @@ TEST_F(Base64Test, calculateDecodedSizeProperSize) {
   EXPECT_EQ(14, encoded_size);
 }
 
+TEST_F(Base64Test, ChecksPadding) {
```

Comment:
test names should start with small case and should indicate the intent eg. `checkPadding`
same below.